index.js

```diff
@@ -2,59 +2,152 @@
 //The whole thing has to be done!
 //Functions can be changed
 //Content needs to be changed.
+	//get n biggest element from the list
+	//i: position of the value in object
+	var getMostNFromArray1= function(array, n, property){
+		var nMost = new Array(n);
+		//sorting the list in decreasing oder
+		array.sort(function(a,b){return b[property] -a[property];});
+		for (var j= 0;j<n;j++){
+			nMost[j]= listA[j];
+		}
+		return nMost;
+	};
+	var getMostNFromArray2= function(array,n, arrayOfProp){
+		var nMost = new Array(n);
+		//sorting the list in decreasing oder
+		array.sort(function(a,b){
+			var aSum = 0,bSum =0;
+			for (var k = 0;k<arrayOfProp.length;k++) {
+				aSum+=a[arrayOfProp[k]];
+				bSum+=b[arrayOfProp[k]];
+			}
+			return aSum -bSum;
+			});
+		for (var j= 0;j<n;j++){
+			nMost[j]= listA[j];
+		}
+		return nMost;
+	};
+	var testDateAndName = function (date,name){
+		var error = new Error();
```

Comment:
to get the elements, whose property is biggest, from the response. For example you want the 3 characters with most "posSum"(topSentiment).
For mostTalkedabout and topControversial its a little difficult, you need the sum from "posCount","negCount","nullCount" for mostTalkedAbout,
"posSum","negSum" for topControversial.
Is it clear now :)